### PR TITLE
Update brandedColleges.js

### PR DESCRIPTION
**Pull request by gmorell**
added an availableRegions for the autocomplete

### CODE REVIEW

File: collections/brandedColleges.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ BrandedColleges.attachSchema(new SimpleSchema({
   aiSubject: {type: String},
   animal: {type: String, optional: true}, // Panther
   askInternet: {type: Boolean, optional: true}, // AH-352
+  availableRegions: {type: [String], optional: true},
```
Comment (AnthonyChiavelli) [inline collections/brandedColleges.js]: Could we make this non-optional and default to []?

Comment (gmorell) [reply collections/brandedColleges.js]: Just for reference, it's optional since only one partner is using the functionality.

**Review by AnthonyChiavelli — APPROVED** [review]

File: collections/brandedColleges.js (hunk 1 above)
Comment (djrochford) [reply collections/brandedColleges.js]: I've been embracing the idea that we should always prefer default values over optional values, the thought being that this approach is less prone to bugs, but I can see why you'd want a field like this to be optional -- interested to hear arguments both ways.

File: README.md [hunk 2]
```diff
@@ -41,6 +41,7 @@ Fields:
  - `oliName` Type: String. Required. Name of the bot personality. (To do: consider changing name. Though maybe we want to keep it for nostalgia resons.)
  - `abbr` Type: String. Optional. Abbreviated form of the institution's name.
  - `animal` Type: String. Optional. The species (or whatever) of the institution's mascot.
+ - `availableRegions` Type: \[String\]. Optional. List of regions to prepopulate into the region picker.
```
Comment (bradddd) [inline README.md:44]: Does this ultimately have filtering/permission implications? e.g. if the `brandedCollege` has `availableRegions` but the individual user doesn't have a `region`, what happens?

Comment (gmorell) [reply README.md:44]: They're considered national, as it works already. This is just to add values to the region picker in the addUser.

**Review by djrochford — COMMENTED** [review]
This all appears in order; interested to hear a bit about the pros and cons of using a default value rather than making it optional before approving.

File: collections/brandedColleges.js (hunk 1 above)
Comment (gmorell) [reply collections/brandedColleges.js]: I don't mind either way. I can set it to be a default to []

**Review by neckenth — APPROVED** [review]
all makes sense to me 👍

**Pull request merged**